openadapt/db/crud.py

```diff
@@ -555,7 +575,9 @@ def get_window_events(
     Returns:
         list[WindowEvent]: A list of window events for the recording.
     """
-    return _get(session, WindowEvent, recording.id)
+    return _get(
+        session, WindowEvent, recording.id, eager=True, relationships=["action_events"]
```

Comment:
Why not `relationships=["action_events", "recording", "screenshot"]`?

Reply:
we can add `recording` but not screenshot